### ISSUE TITLE
Accessibility: Contrast between text and background

### ISSUE THREAD

**Issue by SunderB** [SunderB/sonic-pi-phoenix-website#6]
All themes should at least meet WCAG 2.1 AA standards:
https://www.w3.org/WAI/WCAG21/quickref/#contrast-minimum

> Large Text: Large-scale text and images of large-scale text have a contrast ratio of at least 3:1;
>
>Incidental: Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.
>
> Logotypes: Text that is part of a logo or brand name has no contrast requirement.

The high contrast theme should meet WCAG 2.1 AAA standards: https://www.w3.org/WAI/WCAG21/quickref/#contrast-enhanced

> Large Text: Large-scale text and images of large-scale text have a contrast ratio of at least 4.5:1;
>
> Incidental: Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.
>
> Logotypes: Text that is part of a logo or brand name has no contrast requirement.
